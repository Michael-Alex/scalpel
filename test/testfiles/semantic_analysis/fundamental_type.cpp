char               v01;
unsigned char      v02;
signed char        v03;
bool               v04;
unsigned           v05;
unsigned int       v06;
signed             v07;
signed int         v08;
int                v09;
unsigned short int v10;
unsigned short     v11;
unsigned long int  v12;
unsigned long      v13;
signed long int    v14;
signed long        v15;
long int           v16;
long               v17;
signed short int   v18;
signed short       v19;
short int          v20;
short              v21;
wchar_t            v22;
float              v23;
double             v24;
long double        v25;
void*              v26;

