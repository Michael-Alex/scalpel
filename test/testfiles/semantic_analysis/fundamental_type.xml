<namespace id="n0">
 <variable name="v1">
  <type>
   <fundamental_type type="char"/>
  </type>
 </variable>
 <variable name="v2">
  <type>
   <fundamental_type type="unsigned char"/>
  </type>
 </variable>
 <variable name="v3">
  <type>
   <fundamental_type type="signed char"/>
  </type>
 </variable>
 <variable name="v4">
  <type>
   <fundamental_type type="bool"/>
  </type>
 </variable>
 <variable name="v5">
  <type>
   <fundamental_type type="unsigned int"/>
  </type>
 </variable>
 <variable name="v6">
  <type>
   <fundamental_type type="unsigned int"/>
  </type>
 </variable>
 <variable name="v7">
  <type>
   <fundamental_type type="int"/>
  </type>
 </variable>
 <variable name="v8">
  <type>
   <fundamental_type type="int"/>
  </type>
 </variable>
 <variable name="v9">
  <type>
   <fundamental_type type="int"/>
  </type>
 </variable>
 <variable name="v10">
  <type>
   <fundamental_type type="unsigned short int"/>
  </type>
 </variable>
 <variable name="v11">
  <type>
   <fundamental_type type="unsigned short int"/>
  </type>
 </variable>
 <variable name="v12">
  <type>
   <fundamental_type type="unsigned long int"/>
  </type>
 </variable>
 <variable name="v13">
  <type>
   <fundamental_type type="unsigned long int"/>
  </type>
 </variable>
 <variable name="v14">
  <type>
   <fundamental_type type="long int"/>
  </type>
 </variable>
 <variable name="v15">
  <type>
   <fundamental_type type="long int"/>
  </type>
 </variable>
 <variable name="v16">
  <type>
   <fundamental_type type="long int"/>
  </type>
 </variable>
 <variable name="v17">
  <type>
   <fundamental_type type="long int"/>
  </type>
 </variable>
 <variable name="v18">
  <type>
   <fundamental_type type="short int"/>
  </type>
 </variable>
 <variable name="v19">
  <type>
   <fundamental_type type="short int"/>
  </type>
 </variable>
 <variable name="v20">
  <type>
   <fundamental_type type="short int"/>
  </type>
 </variable>
 <variable name="v21">
  <type>
   <fundamental_type type="short int"/>
  </type>
 </variable>
 <variable name="v22">
  <type>
   <fundamental_type type="wchar_t"/>
  </type>
 </variable>
 <variable name="v23">
  <type>
   <fundamental_type type="float"/>
  </type>
 </variable>
 <variable name="v24">
  <type>
   <fundamental_type type="double"/>
  </type>
 </variable>
 <variable name="v25">
  <type>
   <fundamental_type type="long double"/>
  </type>
 </variable>
 <variable name="v26">
  <type>
   <pointer>
    <fundamental_type type="void"/>
   </pointer>
  </type>
 </variable>
</namespace>
