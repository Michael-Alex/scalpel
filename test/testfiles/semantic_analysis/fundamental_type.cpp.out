<namespace id="0">
 <variable name="v01">
  <type>
   <fundamental_type type="char"/>
  </type>
 </variable>
 <variable name="v02">
  <type>
   <fundamental_type type="unsigned char"/>
  </type>
 </variable>
 <variable name="v03">
  <type>
   <fundamental_type type="signed char"/>
  </type>
 </variable>
 <variable name="v04">
  <type>
   <fundamental_type type="bool"/>
  </type>
 </variable>
 <variable name="v05">
  <type>
   <fundamental_type type="unsigned int"/>
  </type>
 </variable>
 <variable name="v06">
  <type>
   <fundamental_type type="unsigned int"/>
  </type>
 </variable>
 <variable name="v07">
  <type>
   <fundamental_type type="int"/>
  </type>
 </variable>
 <variable name="v08">
  <type>
   <fundamental_type type="int"/>
  </type>
 </variable>
 <variable name="v09">
  <type>
   <fundamental_type type="int"/>
  </type>
 </variable>
 <variable name="v10">
  <type>
   <fundamental_type type="unsigned short int"/>
  </type>
 </variable>
 <variable name="v11">
  <type>
   <fundamental_type type="unsigned short int"/>
  </type>
 </variable>
 <variable name="v12">
  <type>
   <fundamental_type type="unsigned long int"/>
  </type>
 </variable>
 <variable name="v13">
  <type>
   <fundamental_type type="unsigned long int"/>
  </type>
 </variable>
 <variable name="v14">
  <type>
   <fundamental_type type="long int"/>
  </type>
 </variable>
 <variable name="v15">
  <type>
   <fundamental_type type="long int"/>
  </type>
 </variable>
 <variable name="v16">
  <type>
   <fundamental_type type="long int"/>
  </type>
 </variable>
 <variable name="v17">
  <type>
   <fundamental_type type="long int"/>
  </type>
 </variable>
 <variable name="v18">
  <type>
   <fundamental_type type="short int"/>
  </type>
 </variable>
 <variable name="v19">
  <type>
   <fundamental_type type="short int"/>
  </type>
 </variable>
 <variable name="v20">
  <type>
   <fundamental_type type="short int"/>
  </type>
 </variable>
 <variable name="v21">
  <type>
   <fundamental_type type="short int"/>
  </type>
 </variable>
 <variable name="v22">
  <type>
   <fundamental_type type="wchar_t"/>
  </type>
 </variable>
 <variable name="v23">
  <type>
   <fundamental_type type="float"/>
  </type>
 </variable>
 <variable name="v24">
  <type>
   <fundamental_type type="double"/>
  </type>
 </variable>
 <variable name="v25">
  <type>
   <fundamental_type type="long double"/>
  </type>
 </variable>
 <variable name="v26">
  <type>
   <pointer>
    <fundamental_type type="void"/>
   </pointer>
  </type>
 </variable>
</namespace>
